lxd/container_lxc.go

```diff
@@ -8928,6 +8928,12 @@ func (c *containerLXC) removeNetworkFilters(deviceName string, m types.Device) {
 		logger.Error("Failed to clear ip6tables ipv6_filter rules: ", log.Ctx{"container": c.Name(), "device": deviceName, "err": err})
 	}
 
+	// Read current static IP allocation configured from dnsmasq host config (if exists).
+	IPv4, IPv6, err := networkDHCPStaticContainerIPs(m["parent"], c.Name())
+	if err != nil {
+		logger.Error("Failed to remove network filters: ", log.Ctx{"container": c.Name(), "device": deviceName, "err": err})
```

Comment:
I don't think we need the `: ` in this error message, the formatting to split error message from context is already handled by the logger

Reply:
@stgraber OK no problem, will remove.